modules/registry-exposing-default-registry-manually.adoc

```diff
@@ -38,6 +38,13 @@ $ HOST=$(oc get route default-route -n openshift-image-registry --template='{{ .
 $ oc extract secret/$(oc get ingresscontroller -n openshift-ingress-operator default -o json | jq '.spec.defaultCertificate.name // "router-certs-default"' -r) -n openshift-ingress --confirm
 ----
 
+. Move the extracted certificate to the system's trusted CA directory by using the following command:
```

Comment:
typically:
```suggestion
. Move the extracted certificate to the system's trusted CA directory by running the following command:
```

I notice that none of these 6 steps have "by running the following command" - worth making them all compliant/consistent while you're here?